institution/views.py

```diff
@@ -77,25 +78,24 @@ def merge_page_vname(self) -> str:
     def return_quick_init_vname(self) -> str:
         return 'institution:return_quick_init'
 
+    @property
+    def default_order(self) -> str:
+        return 'asc'
+
     def get_queryset(self):
+        if not self.request_data:
+            return CofkUnionInstitution.objects.none()
+
         return self.get_queryset_by_request_data(self.request_data, sort_by=self.get_sort_by())
 
     def get_queryset_by_request_data(self, request_data, sort_by=None) -> Iterable:
         # queries for like_fields
         queries = query_utils.create_queries_by_field_fn_maps(self.search_field_fn_maps, request_data)
 
-        search_fields_maps = {
-            'institution_name': ['institution_name', 'institution_synonyms'],
-            'institution_city': ['institution_city', 'institution_city_synonyms'],
-            'institution_country': ['institution_country', 'institution_country_synonyms'],
-            'resources': ['resources__resource_name', 'resources__resource_details',
-                          'resources__resource_url'],
-            'images': ['images__image_filename']}
-
         queries.extend(
-            query_utils.create_queries_by_lookup_field(request_data, self.search_fields, search_fields_maps)
+            query_utils.create_queries_by_lookup_field(request_data, self.search_fields, self.search_field_combines)
         )
-        return self.create_queryset_by_queries(CofkUnionInstitution, queries, sort_by=sort_by).distinct()
+        return self.create_queryset_by_queries(CofkUnionInstitution, queries, sort_by=sort_by)
```

Comment:
`distinct` can be remove in location query is because it use `exist subquery`. So, if you want to remove `distinct` you have to use `exist subquery`

Reply:
Reverted to use distinct